pkg/services/accesscontrol/ossaccesscontrol/ossaccesscontrol.go

```diff
@@ -0,0 +1,64 @@
+package ossaccesscontrol
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/registry"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/accesscontrol/evaluator"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+// OSSAccessControlService is the service implementing role based access control.
+type OSSAccessControlService struct {
+	Cfg               *setting.Cfg `inject:""`
+	Log               log.Logger
+	builtInRoleGrants map[string][]string
```

Comment:
Why having this struct field since it is a package scope variable already (in builtin_roles.go) ?

Reply:
If its necessary then why don't we add builtInRoles too?